dbt_subprojects/tokens/models/prices_v2/prices_v2_latest.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+        schema='prices_v2',
+        alias = 'latest',
```

Comment:
Why is this model needed?

Reply:
It's not needed for the sqlmesh side, just thought it would be cheaper do do it directly on the sparse dataset then later on the filled one.